apex/contrib/torchsched/ops/layer_norm.py

```diff
@@ -0,0 +1,467 @@
+"""Customized CuDNN frontend layer norm.
+
+Please refer to:
+
+* https://github.com/NVIDIA/cudnn-frontend/blob/main/samples/python/20_layernorm.ipynb
+"""
+
+from __future__ import annotations
+
+import math
+from typing import Sequence
+
+import cudnn
```

Comment:
ideally it'd be nice to check cudnn-frontend availability and version

Reply:
As per discussion, it's fine to not mention python cudnn-frontend for now, because there are a few extensions that depend on cudnn-frontend while they're c++ pytorch extension.

Reply:
oh thanks for putting it here. I really appreciate it